onert-micro/eval-driver/Driver.cpp

```diff
@@ -114,7 +114,7 @@ int entry(int argc, char **argv)
     }
 
     // Do inference.
-    interpreter.run();
+    interpreter.run(config);
```

Comment:
config argument for OMInterpreter is mandatory ?

Reply:
we need this to build for onert-micro-dev : 
```
diff --git a/onert-micro/onert-micro/src/api/onert-micro.cpp b/onert-micro/onert-micro/src/api/onert-micro.cpp
index 538529b32b..37f2d63e61 100644
--- a/onert-micro/onert-micro/src/api/onert-micro.cpp
+++ b/onert-micro/onert-micro/src/api/onert-micro.cpp
@@ -307,7 +307,7 @@ NNFW_STATUS nnfw_session::train_run(bool update_weights)
     float *user_input_data = (float *)_train_interpreter->getInputData(0);
     memcpy(allocated_input_data, user_input_data,
            sizeof(float) * _train_interpreter->getInputSizeAt(0));
-    _train_interpreter->run();
+    _train_interpreter->run(_config);
     float *calculated_ptr = (float *)_train_interpreter->getOutputDataAt(0);
     memcpy(outputbuf, calculated_ptr, sizeof(float) * _train_interpreter->getOutputSizeAt(0));
     _train_interpreter->reset();
@@ -380,7 +380,7 @@ NNFW_STATUS nnfw_session::train_get_loss(uint32_t index, float *loss)
       break;
   }

-  _train_interpreter->evaluateMetric(m, reinterpret_cast<void *>(loss),
+  _train_interpreter->evaluateMetric(_config, m, reinterpret_cast<void *>(loss),
                                      _config.training_context.batch_size);
   return NNFW_STATUS_NO_ERROR;
 }
```

Reply:
> config argument for OMInterpreter is mandatory ?

Yes, this is helpful for running GRU node for further training

Reply:
> we need this to build for onert-micro-dev :

Updated it , thank you :)